problems/euler/Multiples_of_3_and_5/python/MarnieRM/main.py

```diff
@@ -0,0 +1,29 @@
+# 2/18/2020 Marnie M Problem 1
+class Main:
+    @staticmethod  # Can be called without needing to create a main object
+    # This function keeps track of the total sum of multiples of 3 and 5 under 100 in sumTotal
+    # and subtracts the sum of the multiples of 15 so there are no repeats.
+    def threeAndFive():
+        countUp = 0
+        sumTotal = 0
+
+        while countUp + 3 < 1000:
+            countUp += 3
+            sumTotal += countUp  # countUp holds the current largest multiple of 3 and adds it to the total sum
+        countUp = 0
+
+        while countUp + 5 < 1000:
+            countUp += 5
+            sumTotal += countUp
+        countUp = 0
+
+        while countUp + 15 < 1000:
+            countUp += 15
+            sumTotal -= countUp
+

```

Comment:
instead of having each of these in a while loop you could have them all in one! Just a nitpicky thing that would shorten the code.  Additionally, this problem can be done without taking the multiples of 15 out by using the modulus function integrated in python (%). Otherwise looks great!

Reply:
I don't think the modulus solution will work since the number would need to be counted once. Since it is so small I think I will keep it separated out but if I were to ever expand it I will move them into one function.